content/article/why-we-sponsor-metacpan-opencage.md

```diff
@@ -0,0 +1,57 @@
+
+  {
+    "title"       : "Why we sponsor MetaCPAN: OpenCage",
+    "authors"     : ["ed-freyfogle"],
+    "date"        : "2024-07-30T10:35:43",
+    "tags"        : ["opencage", "metacpan", "sponsorship", "geocoding"],
+    "draft"       : true,
```

Comment:
We may as well set this to `false`, as we'll want to publish right after merge anyway.